server/app/auth/CiviFormProfileData.java

```diff
@@ -18,6 +19,8 @@
  */
 public class CiviFormProfileData extends CommonProfile {
 
+  private static final long serialVersionUID = 3142603030317816700L;
```

Comment:
suggestion (blocking): please add some context for why this value in particular and what it is doing

Reply:
Done in parent PR #6063